utils/build/docker/runner.Dockerfile

```diff
@@ -0,0 +1,31 @@
+FROM python:3.9
+
+
+RUN mkdir /app
+WORKDIR /app
+
+COPY requirements.txt .
+COPY build.sh .
+COPY utils/build/build.sh utils/build/build.sh
+RUN mkdir -p /app/utils/build/docker && ./build.sh -i runner
+
+# tests
+COPY tests /app/tests
+COPY parametric /app/parametric
+
+# basically everything except utils/build
+COPY utils/_context /app/utils/_context
+COPY utils/assets /app/utils/assets
+COPY utils/grpc /app/utils/grpc
+COPY utils/proxy /app/utils/proxy
+COPY utils/interfaces /app/utils/interfaces
+COPY utils/scripts /app/utils/scripts
+COPY utils/test_the_test /app/utils/test_the_test
+COPY utils/*.py /app/utils/
```

Comment:
This section attempts to minimize dependencies when building the image. In this case `utils/build` is entirely irrelevant but it can't be added to `.dockerignore` since the latter has its own docker things.

A bit of refactoring with dedicated folders might make that easier.